skunkworks_crow/src/main/java/org/odk/share/activities/SendActivity.java

```diff
@@ -88,9 +87,6 @@
     private long[] formIds;
     private int mode;
 
-    private WifiManager.LocalOnlyHotspotReservation hotspotReservation;
```

Comment:
@shobhitagarwal1612 Is there any specific reason for removing this class and its API usage? I think Google has introduced this Class and some APIs so that hotspot connection can be made and two nearby devices can share the data locally without sharing the user's mobile data.